inventory/publish-docs/group_vars/all.yml

```diff
@@ -0,0 +1,21 @@
+---
+
+confluence:
```

Comment:
Looking at the overall implementation, we may want to scope this with a parent variable - i.e.: `publish-docs`, so in that way the dictionary will be something like:

```
publish-docs:
  confluence:
    username: ...
      :
  github:
    repo_owner: ...
     :
```